tests/test_backends.py

```diff
@@ -188,3 +189,15 @@ def test_superposition_for_all_qubits(connected_backend):
 
 # TODO: test_circuit_result_tensor
 # TODO: test_circuit_result_representation
+
+
+def test_metabackend_load(dummy_qrc):
+    for platform in Path("tests/dummy_qrc/").iterdir():
+        backend = MetaBackend.load(platform.name)
+        assert isinstance(backend, QibolabBackend)
+        assert Path(backend.platform.name).name == platform.name
+
+
+def test_metabackend_list_available(dummy_qrc):
+    available_platforms = {d.name: True for d in Path("tests/dummy_qrc/").iterdir()}
+    assert MetaBackend().list_available() == available_platforms
```

Comment:
This test is a bit funny because it is almost copying the implementation. We are probably doing the same in other places in qibolab and other qibo repos, just noticed it here because of the simplicity (the `dict` comprehension and use of `iterdir` is identical). I think a more proper way of testing uses specific examples/edge cases, but I am also not an expert on testing. Btw, related discussion: https://github.com/qiboteam/qibolab/pull/786#discussion_r1474573320